src/app/features/bitcoin-choose-fee/components/choose-fee-tabs.tsx

```diff
@@ -15,40 +18,34 @@ interface ChooseFeeTabsProps extends StackProps {
 export function ChooseFeeTabs(props: ChooseFeeTabsProps) {
   const { feesList, customFee, ...rest } = props;
   const analytics = useAnalytics();
-  // TODO #4013: Refactor this to use routes for tabs like home-tabs
```

Comment:
I was seeing some [Sentry errors](https://trust-machines.sentry.io/issues/4848769501/?alert_rule_id=14461919&alert_type=issue&notification_uuid=467bc53c-2730-4d31-b021-83588e2c34c4&project=4504204000952320&referrer=slack ) come in about `setActiveTabTracked`. Did you see those?

Reply:
Hadn't seen, but now I've removed this completely?